addons/api/addon/models/user.js

```diff
@@ -40,7 +40,7 @@ export default class UserModel extends GeneratedUserModel {
    * There are many options, so we fallback in order of precedence.
    * @type {string}
    */
-  @computed('email,full_name,login_name,displayName')
+  @computed('email', 'full_name', 'login_name')
```

Comment:
This computed decorator may not be required at all in Ember Octane:  https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-computed-properties-in-native-classes.md

It should automagically work the same if the decorator is removed thanks to tracked properties (model attributes are automatically tracked).